Makefile

```diff
@@ -52,7 +60,16 @@ verify: tidy generate format
 	git diff --exit-code
 
 install: generate
-	kubectl apply -f manifests
+	kubectl apply -f manifests/crds && kubectl apply -f manifests/deploy
+
+install-fresh: generate load-container install
+
+uninstall:
+	kubectl delete -f manifests/deploy && kubectl delete -f manifests/crds
+
+reinstall: uninstall install
+
+reinstall-fresh: uninstall install-fresh
```

Comment:
I personally don't see a ton of value including these types of targets but I'm open to opinions that have a preference towards a reinstall-type target.

Reply:
I mainly like reinstall types of targets since you just run one command to get everything related to the application up-to-date. That being said, I'm not strongly opinionated here, it was something I was using a lot while developing to make sure everything worked together but I could be convinced to not have it included in the repo.